web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -246,7 +246,35 @@ const sfc = {
         event.preventDefault();
         this.menuVisible = false;
       }
-    }
+    },
+    handleTabSwitch(mode) {
+      this.createTag = mode === 'tags';
```

Comment:
The `createTag` can be computed?

Reply:
Moved to computed https://github.com/go-gitea/gitea/pull/25719/files#diff-588b4b6c148c056a96c4cb1732bb5a12b12ee31ed2e2938eaf751b44e5f9fd64